package.json

```diff
@@ -17,9 +17,26 @@
   },
   "homepage": "https://github.com/in-need-org/hrtz.io#readme",
   "dependencies": {
-    "express": "3.4.4"
+    "@nivo/stream": "^0.50.0",
+    "@types/lodash": "^4.14.117",
+    "@types/moment": "^2.13.0",
+    "@types/react": "^16.4.18",
+    "@types/react-dom": "^16.0.9",
+    "awesome-typescript-loader": "^5.2.1",
+    "express": "^4.16.4",
+    "highcharts-react-official": "^1.5.1",
```

Comment:
```suggestion
```

Reply:
actually it seems that your latest commit was to your branch rather than master, since i remember you removed this and some other packages